FakeTrello/README.md

```diff
@@ -0,0 +1,20 @@
+# Fake Trello
+
+## Features
+
+- Users will be able to create boards, lists and cards.
+- Users will be able to delete baords, lists and cards.
+- Users will be able to attach up to 3 other users to a cards (regardless of card creator).
+- Users will be able to move and copy Cards between lists
+
+
+## Instructions
+
+1. Create a Logical ERD (models with columns).
+2. Add your ERD to this README as an image. (under the "Fake Trello ERD" section)
+2. Create C#/Entity models using your ERD
+
+
+### Fake Trello ERD
+
+![My ERD](/FakeTrello.jpg?raw=true)
```

Comment:
oh, mmmm I added the image to mine by editing the README on Github....I may need to import the image into my project folder.  I can refactor my code to match my ERD.